python/psautohint/autohint.py

```diff
@@ -294,7 +297,8 @@ def save(self, path):
                 fp.write(header)
                 reps.sort(key=sortFunc)
                 for item in reps:
```

Comment:
I realize this part of the code predates this change, but here at line 299 we have the start of a `for item in ...` loop that is *inside* another `for i, item in ...` loop starting at line 288. I suggest changing at least one of them so we're not using `item` in both places.

This is just a suggestion. Since it seems to work, and since it predates this PR, I'm going to go ahead and approve this PR.

Reply:
good catch. Changed.